jbmc/regression/jbmc/classpath-class-with-jar/test.desc

```diff
@@ -1,7 +1,8 @@
 CORE
-Test.class
+Test
 --classpath `../../../../scripts/format_classpath.sh src.jar .`
 ^EXIT=10$
+^EXIT=10$
```

Comment:
Changing the tests at the same time as changing how they are opened makes this much harder to review

Reply:
That's a superfluous change, actually.